userspace/libsinsp/threadinfo.cpp

```diff
@@ -515,8 +515,22 @@ void sinsp_threadinfo::set_args(const char* args, size_t len)
 
 void sinsp_threadinfo::set_env(const char* env, size_t len)
 {
-	m_env.clear();
+	if (len == SCAP_MAX_ENV_SIZE && m_inspector->is_live())
```

Comment:
Should this be >= instead of == ?

Reply:
If we truncate the environment, len will be equal to `SCAP_MAX_ENV_SIZE`.  So >= is correct as well but not necessary. Currently, `len > SCAP_MAX_ENV_SIZE` is not possible and I'm not sure what would be the correct way to handle it.